### ISSUE TITLE
hs.eventtap.keyStrokes only send aaaa with RDP

### ISSUE THREAD

**Issue by JonLur** [Hammerspoon/hammerspoon#2881]
I try to use hs.eventtap.keyStrokes  function with Microsoft Remote Desktop Beta 10.6.7 and it only sendes "aaaaa"
This is not a new problem. It has never worked.
Not sure if this problem is Hammerspoon or Microsoft Remote Desktop

Code used:
hs.hotkey.bind({"ctrl", "alt"}, "V", "Copy pasteboard", function()
     teststring=hs.pasteboard.getContents()
     hs.printf(teststring)
     hs.eventtap.keyStrokes(teststring)
end)

* hs.printf always print correct string in Hammerspoon Console
* In RDP window it only shows "aaaaaaaaaaaa" - the number of "a" is the same as the string length of teststring. 
* Normal copy / paste is working correctly in RDP window
* I want to use this in password field where copy/paste does not work.

**Comment by andrewcrook** [comment]
Have you tried creating an hs application object for the RDP client app then pass that as a parameter keyStrokes?

`hs.eventtap.keyStrokes(text[, application])`

`application - An optional hs.application object to send the keystrokes to`

**Comment by JonLur** [comment]
I tried your proposal - and no difference.

However - I did some more testing and inside Microsoft Remote Desktop there is a setting for "Keyboard Mode" and this was set to "Scancode". I changed it to "Unicode" and then the problem was solved.
I do fully not understand the difference between these two settings.

I remeber that I had some keyboard problems long time ago which was solved by change this setting to "Scancode". Do not remember what kind of problem.
